README.md

```diff
@@ -16,7 +16,7 @@ in OpenStreetMap.
 ### Browse the index
 
 You can browse the index at
-http://osmlab.github.io/name-suggestion-index/brands/index.html
+http://osmlab.github.io/name-suggestion-index/brands
```

Comment:
this is a 404

Reply:
Yeah I see that now - gh-pages isn't quite compatible with react router, so I need to change a bit how the app works.  I'll fix it tonight..

Reply:
It works now!  I switched from using path-based routing to using querystrings instead..

Reply:
It appears that clicking on any of the links just loops back to the main brand page.

Reply:
Thanks for reporting @Adamant36  -  I fixed it in 2ec4117f0, but let me know if you find any other issues.